src/global.ts

```diff
@@ -118,6 +118,14 @@ interface RedirectLoginResult {
 
 interface PopupLoginOptions extends BaseLoginOptions {}
 
+interface PopupConfigOptions {
+  /**
+   * The number of seconds to wait for a popup response before
+   * throwing a timeout error
```

Comment:
Please add the default value, or point to the constant name for the user to check if you think this will change often.